jobs/ci-kubernetes-soak-cos-docker-validation-deploy.env

```diff
@@ -1,5 +1,8 @@
 KUBE_OS_DISTRIBUTION=gci
 
+# Use fixed k8s version.
+JENKINS_PUBLISHED_VERSION=v1.6.1
+
 ### soak-env
```

Comment:
Yes. And if you do that, you can even set `JENKINS_USE_GCI_VERSION=y`. That way you don't have to hard code the k8s version and the infra will figure it out from the COS image version.